server/models/User.model.ts

```diff
@@ -28,6 +30,28 @@ class User extends Model<InferAttributes<User>, InferCreationAttributes<User>> {
       'website',
     ];
   }
+
+  public static associations: {
+    flags: Association<User, Flag>;
+  };
+
+  public static associate() {
+    User.hasMany(Flag, { foreignKey: 'user_id' });
+  }
```

Comment:
The docs mention we also need a `Flag.belongsTo(User)` otherwise only the `User` model knows about the association while the `Flag` model does not

based on this example
```js
Team.hasMany(Player);
Player.belongsTo(Team);
```

I'm confused about the example where this is supposed to go though but I think after `User.init` step because its shown like that in another [example](https://sequelize.org/docs/v6/advanced-association-concepts/creating-with-associations/#belongsto--hasmany--hasone-association)



https://sequelize.org/docs/v6/core-concepts/assocs/#implementation-1
https://sequelize.org/docs/v6/core-concepts/assocs/#why-associations-are-defined-in-pairs

Reply:
Based on voice chat, Devon's & Tim's comments were as it stands we don't need to be able to search users from the Flag models so a `Flag.belongsTo(Users)` isn't nesccarry